objects/src/constants.rs

```diff
@@ -2,7 +2,7 @@
 pub const ACCOUNT_TREE_DEPTH: u8 = 64;
 
 /// The depth of the Merkle tree used to commit to notes produced in a block.
-pub const NOTE_TREE_DEPTH: u8 = 20;
+pub const OUTPUT_NOTE_TREE_DEPTH: u8 = 20;
```

Comment:
I'm not sure we need to do this (and other related) renaming. The note tree in a block contains just notes - these are output notes at some point and input notes at another point. Also, there isn't really a concept of an "input note tree" - so, just "note tree" is unambiguous, I think.